scripts/load-breaches.js

```diff
@@ -11,19 +11,19 @@ const pkg = require("../package.json");
 
 const HIBP_USER_AGENT = `${pkg.name}/${pkg.version}`;
 
-
 const DOMPurify = createDOMPurify((new JSDOM("")).window);
 
-const args = arg({
-  "--createAMBreach": Boolean,
-  "--help": Boolean,
-});
+// const args = arg({
+//   "--createExample": Boolean,
+//   "--help": Boolean,
+// });
 
-if (args["--help"]) {
-  console.log("Usage: node load-breaaches.js [--createAMBreach]");
-  console.log("--createAMBreach creates the 'AllMusic' test fixture breach.");
-  process.exit();
-}
+// if (args["--help"]) {
+//   console.log("\n\n  Usage: node load-breaches.js [--createExample]");
+//   console.log("  --createExampleBreach creates an example breach.");
```

Comment:
nit: I think this should be `--createExample` and not `--createExampleBreach`

Also, I think there are a couple more hidden references to AllMusic in **/scripts/add-breached-emails.js**:

1. https://github.com/mozilla/blurts-server/blob/fe08c44b25d13ac34841bd29a977860e6a249c37/scripts/add-breached-emails.js#L17-L18
1. https://github.com/mozilla/blurts-server/blob/fe08c44b25d13ac34841bd29a977860e6a249c37/scripts/add-breached-emails.js#L31-L34